sdk/ai/azure-ai-projects/azure/ai/projects/models/_patch.py

```diff
@@ -1314,17 +1314,17 @@ def get_last_message_by_sender(self, sender: str) -> Optional[ThreadMessage]:
                 return msg
         return None
 
-    def get_last_text_message_by_sender(self, sender: str) -> Optional[MessageTextContent]:
-        """Returns the last text message from the specified sender.
+    def get_last_text_message_by_role(self, role: MessageRole) -> Optional[MessageTextContent]:
```

Comment:
Thanks for updating this. could you please update also get_last_message_by_sender method ?

Reply:
done